framework/auth/oauth_scopes.py

```diff
@@ -124,8 +128,8 @@ class ComposedScopes(object):
     NODE_ALL_WRITE = NODE_ALL_READ + NODE_METADATA_WRITE + NODE_DATA_WRITE + NODE_ACCESS_WRITE
 
     # Full permissions: all routes intended to be exposed to third party API users
-    FULL_READ = NODE_ALL_READ + USERS_READ + ORGANIZER_READ + GUIDS_READ + (CoreScopes.INSTITUTION_READ, )
-    FULL_WRITE = NODE_ALL_WRITE + USERS_WRITE + ORGANIZER_WRITE + GUIDS_READ
+    FULL_READ = NODE_ALL_READ + USERS_READ + ORGANIZER_READ + GUIDS_READ + (CoreScopes.INSTITUTION_READ, CoreScopes.ADDONS_READ, )
+    FULL_WRITE = NODE_ALL_WRITE + USERS_WRITE + ORGANIZER_WRITE + GUIDS_READ + (CoreScopes.NODE_ADDON_WRITE, )
```

Comment:
We've been requested to consider, among other use cases, allowing someone to make an application that will allow a user to pick a github repo they control and, with minimal clicking, create an OSF user, link the github account to their OSF account, and create a project with that github repo attached. Possibly some other flourishes in addition.

That being said, it is true that said scope could allow an app to link to, say, the root of someone's dropbox, create a project, archive all the info from that dropbox into the new project, add another user to the project, and remove the authorizing user from that project. So we might want to have a process in place for blessing apps that do that kind of thing.